packages/teams-js/test/public/meeting.spec.ts

```diff
@@ -23,142 +23,228 @@ describe('meeting', () => {
   });
 
   describe('toggleIncomingClientAudio', () => {
+    const allowedContexts = [FrameContexts.sidePanel, FrameContexts.meetingStage];
     it('should not allow calls before initialization', () => {
       expect(() => meeting.toggleIncomingClientAudio()).toThrowError('The library has not yet been initialized');
     });
 
-    it('should successfully toggle the incoming client audio', async () => {
-      await desktopPlatformMock.initializeWithContext('sidePanel');
+    Object.values(FrameContexts).forEach(context => {
+      if (allowedContexts.some(allowedContext => allowedContext === context)) {
+        return;
+      }
+      it(`should not allow meeting.toggleIncomingClientAudio calls from ${context} context`, async () => {
+        expect.assertions(1);
+        await utils.initializeWithContext(context);
 
-      const promise = meeting.toggleIncomingClientAudio();
-
-      const toggleIncomingClientAudioMessage = desktopPlatformMock.findMessageByFunc('toggleIncomingClientAudio');
-      expect(toggleIncomingClientAudioMessage).not.toBeNull();
-      const callbackId = toggleIncomingClientAudioMessage.id;
-      desktopPlatformMock.respondToMessage({
-        data: {
-          id: callbackId,
-          args: [null, true],
-        },
-      } as DOMMessageEvent);
-      await expect(promise).resolves.toBe(true);
+        expect(() => meeting.toggleIncomingClientAudio()).toThrowError(
+          `This call is only allowed in following contexts: ${JSON.stringify(
+            allowedContexts,
+          )}. Current context: "${context}".`,
+        );
+      });
     });
 
-    it('should return error code 500', async () => {
-      await desktopPlatformMock.initializeWithContext('meetingStage');
+    Object.values(FrameContexts).forEach(context => {
+      if (!allowedContexts.some(allowedContext => allowedContext === context)) {
+        return;
+      }
```

Comment:
See my comment above about changing the if statement and not using `return`

Reply:
Should we follow the same pattern in other files like mail.spec or should we leave them as they are already done?